ewlegal-plead.sty

```diff
@@ -0,0 +1,58 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+\ProvidesPackage{engwales-plead}[2021-02-13 England and Wales Pleading Legal Template]
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Variables
+%%%%%
+
+%% For determining text in the title
+\newcommand{\court}[1]{\def\@court{#1}}
+\newcommand{\casenumber}[1]{\def\@casenumber{#1}}
+\newcommand{\claimant}[1]{\def\@claimant{#1}}
+\newcommand{\defendant}[1]{\def\@defendant{#1}}
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typeset Elements
+%%%%%
+
+%% Format the title
+\renewcommand\maketitle{%
+  \begingroup
+  \linespread{1.5}\setlength\parindent{0pt}\bfseries
+  \parbox[t]{.65\textwidth}{\uline{\@court}}
+  Case No~\@casenumber
+
+  \vspace{24pt}
+
+  \so{BETWEEN}
+  \begin{center}\@claimant\end{center}
+  \begin{flushright}\uline{Claimant}\end{flushright}
+  \begin{center}and\end{center}
+  \begin{flushright}\end{flushright}
+  \begin{center}\@defendant\end{center}
+  \begin{flushright}\uline{Defendant}\end{flushright}
+
+  \vspace{0pt}
+
+  \begin{center}
+    \tramlined{\@title}
+  \end{center}
+  \vspace{24pt}
+  \endgroup
+}
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Typesetting
+%%%%%
+
+%% Headers and Footers
+\fancyhf{} %% Clears all fields
+\renewcommand{\headrulewidth}{0pt} %% Removes the header rule
+\rfoot{\thepage}
+\pagestyle{fancy}
```

Comment:
this part is the same as in ewlegal-opinion. extract to .cls for now.

Reply:
I don't believe @passamo9 has given clear indication on what header/footer elements should exist so I separated them in case they were different. If @passamo9 could indicate if there is any difference in the headers and footers depending on document type, that'd be useful for correct placement.